scripts/validate_config.py

```diff
@@ -102,6 +103,14 @@ def confirm_submission_privkey_file(self):
     def confirm_submission_privkey_fingerprint(self):
         assert "submission_key_fpr" in self.config
         assert re.match("^[a-fA-F0-9]{40}$", self.config["submission_key_fpr"])
+        gpg_cmd = ["gpg2", "--show-keys", self.secret_key_filepath]
+        try:
+            out = subprocess.check_output(gpg_cmd).decode(sys.stdout.encoding)
```

Comment:
`check_output` defaults to `stderr=None`. Errors printed to `STDERR` by `gpg2` will therefore not be passed along for the exception handler below. To combine them, you could use `stderr=subprocess.STDOUT` here, or you could capture it via `stderr=subprocess.PIPE` and access it via `e.stderr`.